packages/bot-web-ui/src/components/dashboard/tutorial-tab/faq-content.tsx

```diff
@@ -38,6 +38,38 @@ const FAQContent = observer(({ faq_list, hide_header = false }: TFAQContent) =>
 
     const { faq_search_value } = dashboard;
 
+    const timer_id = React.useRef<ReturnType<typeof setTimeout> | null>(null);
+
+    const handleAccordionClick = () => {
+        // Scroll to the top of the open accordion item.
+        // Need timer to first close the accordion item then scroll the new item to top.
+        timer_id.current = setTimeout(() => {
+            const faq_wrapper_element = document?.querySelector('.faq__wrapper');
```

Comment:
Can we `useRef` and do this on the parent component? Feels wrong to select this on the `document`. What if there's multiple components?

Reply:
Yes @markwylde-deriv, for the faq__wrapper we can use the useRef as it is from the same component and that would be better definitely. There's another selector from the document which is the currently opened accordion, for that it would be difficult to reference using useRef as the accordion is coming from another package. I am updating this faq__wrapper.

Reply:
Done ✅